UnityProject/Assets/Scripts/HealthV2/Living/LivingHealthMasterBase.cs

```diff
@@ -0,0 +1,370 @@
+	/// <param name="damage">Damage Amount</param>
+	/// <param name="attackType">type of attack that is causing the damage</param>
+	/// <param name="damageType">The Type of Damage</param>
+	/// <param name="bodyPartAim">Body Part that is affected</param>
+	[Server]
+	public virtual void ApplyDamageToBodypart(GameObject damagedBy, float damage,
+		AttackType attackType, DamageType damageType, BodyPartType bodyPartAim)
+	{
+		//TODO: Reimplement
+	}
+
+	private void TryGibbing(float damage)
+	{
+		//TODO: Reimplement
+	}
+
+	/// <summary>
+	///  Apply healing to a living thing. Server Only
+	/// </summary>
+	/// <param name="healingItem">the item used for healing (bruise pack etc). Null if there is none</param>
+	/// <param name="healAmt">Amount of healing to add</param>
+	/// <param name="damageType">The Type of Damage To Heal</param>
+	/// <param name="bodyPartAim">Body Part to heal</param>
+	[Server]
+	public virtual void HealDamage(GameObject healingItem, int healAmt,
+		DamageType damageTypeToHeal, BodyPartType bodyPartAim)
+	{
+		//TODO: Reimplement
+	}
+
+	[Server]
+	public void Harvest()
+	{
+
+		//Reimplement
+
+		Gib();
+	}
+
+	[Server]
+	protected virtual void Gib()
+	{
+
+		//TODO: Reimplement
+
+		//never destroy players!
+		Despawn.ServerSingle(gameObject);
+	}
+
+	/// ---------------------------
+	/// CRIT + DEATH METHODS
+	/// ---------------------------
+	///Death from other causes
+	public virtual void Death()
+	{
+		//TODO: Reimplemenmt
+	}
+
+	public virtual void AddSickness(Sickness sickness)
+	{
+		//TODO: Reimplement
+	}
+
+	public void UpdateClientHealthStats(float overallHealth)
+	{
+		//TODO: Reimplement
+	}
+
+	/// <summary>
+	/// Updates the conscious state from the server via NetMsg
+	/// </summary>
+	public void UpdateClientConsciousState(ConsciousState proposedState)
+	{
+		ConsciousState = proposedState;
+	}
+
+	/// <summary>
+	/// Updates the respiratory health stats from the server via NetMsg
+	/// </summary>
+	public void UpdateClientRespiratoryStats(bool value)
+	{
+		respiratorySystem.IsSuffocating = value;
+	}
+
+	public void UpdateClientTemperatureStats(float value)
+	{
+		respiratorySystem.temperature = value;
+	}
+
+	public void UpdateClientPressureStats(float value)
+	{
+		respiratorySystem.pressure = value;
+	}
+
+	/// <summary>
+	/// Updates the blood health stats from the server via NetMsg
+	/// </summary>
+	public void UpdateClientBloodStats(int heartRate, float bloodVolume, float oxygenDamage, float toxinLevel)
+	{
+		//TODO: Reimplement bloodSystem.UpdateClientBloodStats(heartRate, bloodVolume, oxygenDamage, toxinLevel);
+	}
+
+	/// <summary>
+	/// Updates the brain health stats from the server via NetMsg
+	/// </summary>
+	public void UpdateClientBrainStats(bool isHusk, int brainDamage)
+	{
+		//TODO: Reimplement
+	}
+
+	public void ChangeFireStacks(float deltaValue)
+	{
+		SyncFireStacks(fireStacks, fireStacks + deltaValue);
+	}
+
+	private void SyncFireStacks(float oldValue, float newValue)
```

Comment:
Codacy found an issue: [Remove this unused method parameter 'oldValue'.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6324510)

Reply:
Was told to ignore this by @ThatDan123 due to sync vars